mautic/mautic
# Refactor onCampaignTriggerActionSendEmailToUser

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8058
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
https://github.com/mautic/mautic/issues/8058

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create campaign with action Send email to user
3. Trigger campaign and check if email was sent to user properly
